### PR TITLE
Workshop a11y: New version

### PR DESCRIPTION
**Pull request by sandrina-p**

### CODE REVIEW

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 1]
```diff
@@ -203,174 +550,348 @@
...
+  </header>
+
+  <div class="cta-sticky">
+    <!-- <button class="u-link">Coming soon!</button> -->
+  </div>
+
+  <span class="t-separator"></span>
+
+  <p class="t-p">
+    Let’s be honest, as web creators, we are still leaving accessibility as an afterthought. Learning how to create web apps using modern UI libraries sounds way more exciting, <i>I understand you...</i>
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:582]: Let’s be honest, as web creators, we are still leaving accessibility as an afterthought. Learning how to create web apps using modern UI libraries sounds way more exciting, <i>I understand you, but...</i>

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 2]
```diff
@@ -203,174 +550,348 @@
...
+  </p>
+
+  <h2 class="t-hook">
+      A11Y doesn't need to be boring
+  </h2>
+  
+    
+  <p class="t-p">
+    Perhaphs you already know that accessibility matters  and you are aware that color contrast is a thing.
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:591]: Perhaps you already know that accessibility matters and you are aware that color contrast is a thing.

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 3]
```diff
@@ -203,174 +550,348 @@
...
+  <p class="t-p">
+    But... <i>what else?</i>
   </p>
 
-  <p class="p">
-    Hi there, I'm
-    <a href={SITE_URL} rel="noreferrer" class="u-link">Sandrina Pereira</a>
-    and I believe that making the web accessible is our duty as web creators.
+  <p class="t-p">
+    In your solo journey into learning accessibility, you’ve might have visited the official
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:599]: In your solo journey into learning accessibility, you might have visited the official

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 4]
```diff
@@ -203,174 +550,348 @@
...
+    In your solo journey into learning accessibility, you’ve might have visited the official 
+    <a href="https://www.w3.org/WAI/" class="u-link">W3C WAI</a>
+    website and took a look at the
+    <a href="https://www.w3.org/TR/WCAG21" class="u-link">
+    WCAG 2.1 guidelines.
+    </a>
+  </p>
+
+  <p class="t-p">
+    But then you felt overwhelmed, lost,... and left it aside.
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:608]: But then you felt overwhelmed, lost...and left it aside.

(according to the internet: The comma should not be used before the ellipsis as a comma is used to indicate a short pause, whereas an ellipsis indicates a longer pause trailing off into nothingness. Thus the comma is redundant)

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 5]
```diff
@@ -203,174 +550,348 @@
...
+  </p>
+
+  <h2 class="t-hook">
+      Let me simplify it for you
+  </h2>
+
+  <p class="t-p">
+    Let me show you why accessibility is worthy of being part of your skillset foundations, and how it benefits way more people that you judge, including you and me.
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:616]: ...and show you why accessibility is worthy of being part of your skillset foundations, and how it benefits way more people than you imagine, including you and me.

Accessibility does not limit your solutions, on contrary, it makes them more inclusive.

(personally eu acho que fica melhor não repetir o "let me" e fazer frases mais declarativas, tem mais power e soa mais confiante. mas não estava errado)

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 6]
```diff
@@ -203,174 +550,348 @@
...
   </p>
 
-  <p class="f-title subtitle hithere">
-    The web is awesome and everyone should be able to enjoy it.
+
+  <p class="t-p">
+    Let me teach you how accessibility does not limit your solutions,
+    on contrary, it makes them more inclusive.
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:622]: (ver a cima)

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 7]
```diff
@@ -203,174 +550,348 @@
...
 
-  <p class="p">
-    This workshop will be packed with little big discoveries! We will explore every common
-    accessibility no-nos and how to make them inclusive for as many people as possible using a mouse
-    or a keyboard.
+  <h2 class="t-title asH2">
+      The workshop
+  </h2>
+  <p class="t-p">
+    This workshop will be packed with little big discoveries! We will explore every common accessibility no-nos and how to make them inclusive for as many people as possible using a mouse or a keyboard.
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:629]: This workshop will be packed with little big discoveries! We will explore every common accessibility no-nos and how to make them more inclusive for as many people as possible using a mouse or a keyboard.

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 8]
```diff
@@ -203,174 +550,348 @@
...
+    {/each}
+  </ul>
+
+
+  <h3 class="t-title asH3">
+    By the end, you will:
+  </h3>
+  <ul class="t-list asCheck">
+    <li>Realize how web accessibility benefits everyone</li>
+    <li>Understand WCAG principles and how is organized</li>
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:667]: Understand WCAG principles and how they're organized

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 9]
```diff
@@ -203,174 +550,348 @@
...
-    Each challenge introduces a new topic, followed by a hands-on exercise for you to apply the
-    concepts learned.
+  <h3 class="t-title asH3">
+    Workshop dynamics
+  </h3>
+  <p class="t-p">
+    In each challenge, I’ll introduce you to a new topic in a simplified way, along with <strong>detailed resources</strong>. 
+  </p>
+  <p class="t-p">
+    Then, you'll have a <strong>hands-on exercise</strong> for you to apply the concepts learned. You can solve it by yourself or by collaborating with a group of 2-3 people.
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:685]: Then, there will be a <strong>hands-on exercise</strong> for you to apply the concepts learned. You can solve it by yourself or by collaborating with a group of 2-3 people.

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 10]
```diff
@@ -203,174 +550,348 @@
...
   </p>
-  <p class="p">
-    After each exercise, we'll go through the solution together and I will clarify any question that
-    you may have.
+  <p class="t-p">
+    Afterwards, we go through the <strong>solution</strong> together, and I’ll <strong>clarify the questions</strong> that you might have.
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:688]: Afterwards, we go through the <strong>solution</strong> together, and I’ll <strong>clarify any questions</strong> that you might have.

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 11]
```diff
@@ -203,174 +550,348 @@
...
-    <li>A modern browser installed: Chrome or Firefox are recommended;</li>
+  <h3 class="t-title asH3">
+    Who is this workshop for?
+  </h3>
+  <p class="t-p">
+    Web developers are the main audience, but if you are a designer,
+    or a QA expert then this workshop will be valuable for you as well. 
+  </p>
+  <p class="t-p">
+    The topics will be explained with beginners in mind and it gets more advanced as we go through it.
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:711]: The topics will be explained with beginners in mind, but the workshop gets more advanced as we go through it.

(assim se o que fica avançado é o workshop. acho que é preciso msm dizer workshop again pq o sujeito desta frase são os topics, e fica a parecer um erro gramatical)

The topics will be explained with beginners in mind, and they will get more advanced as we go through them.

(assim se o que fica avançado são os tópicos)

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 12]
```diff
@@ -203,174 +550,348 @@
...
+  </p>
+  <p class="t-p">
+    Even if you already have some experience into A11Y, you can look at this workshop as a way to solidify your knowledge and fill any existing gap.
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:714]: Even if you already have some experience in A11Y, you can look at this workshop as a way to solidify your knowledge and fill any existing gaps.

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 13]
```diff
@@ -203,174 +550,348 @@
...
+  </p>
+
+  <h3 class="t-title asH3">
+    Pre-requisites
+  </h3>
+  <ul class="t-list">
+    <li>Comfortable with HTML and CSS. Basics of JavaScript is a bonus;</li>
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:721]: Comfortable with HTML and CSS (knowing the basics of JavaScript will be a bonus);

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 14]
```diff
@@ -203,174 +550,348 @@
...
+    <li>A <strong>modern browser</strong> installed: Chrome or Firefox are recommended;</li>
     <li>
-      A good internet connection with
+      A <strong>good internet connection</strong> with
       <a class="u-link" href="http://zoom.com/" rel="noreferrer">Zoom</a>
       installed for the video call;
     </li>
+    <li>
+      A webcam is optional. Although I highly appreacite seeing you while I’m speaking, I want to be respectful of your own space.
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:729]: A webcam is optional - although I highly appreciate seeing you while I’m speaking, I want to be respectful of your boundaries.

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 15]
```diff
@@ -203,174 +550,348 @@
...
+    F.A.Q.
+  </h2>
+
+  <div class="faqArea">
+    <ul class="faqList">
+    <li>
+        <Accordion summary="What does A11Y mean?">
+          A11Y stands for 'accessibility'.
+          The irony about the word "accessibility" itself is that it's not very easy to pronounce.
+          So, it was created this abbreviation: A + 11 characters + Y, which has a more friendly homophone of "ally".
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:795]: The abbreviation stands for A + 11 characters + Y, which is a more user-friendly homophone of "ally".

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 16]
```diff
@@ -203,174 +550,348 @@
...
+          No, it won't be recorded, but you’ll have access to all the materials shared.
+        </Accordion>
+      </li>
+
+
+      <li>
+        <Accordion summary="What materials are provided?">
+         <ul class="t-list">
+          <li>
+            The materials slides (+ 50 slides);
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:810]: The materials' slides (+ 50 slides);

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 17]
```diff
@@ -203,174 +550,348 @@
...
+          </li>
+          <li>
+            Codebase with all the mentioned resources;
+          </li>
+          <li>
+            Exercises, including the solutions explained;
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:816]: Exercises and their solutions explained;

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 18]
```diff
@@ -203,174 +550,348 @@
...
+          </li>
+         </ul>       <!-- TODO list -->
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="I know A11Y. Will I learn something new?">
+          Oh, yeah. Even me, every time I give this workshop I always learn something new. Unless you are an A11Y expert, I’m confident that you'll fill some gap.
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:824]: Oh, yeah. Personally, every time I give this workshop I always learn something new. Unless you are an A11Y expert, I’m confident that you'll fill some gaps.

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 19]
```diff
@@ -203,174 +550,348 @@
...
+        <Accordion summary="Is the workshop accessible?">
+          The video meeting can be captioned and the exercises briefings are fully accessible. If you have any accessibility concern, please reach out to me, and we can arrange it together.
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="What screen reader will I use?">
+                        <!-- TODO list -->
+
+          You can use the one that suits you the best. I own a Mac, which means I’ll be using Voice Over.
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:838]: You can use the one that suits you best. I own a Mac, which means I’ll be using Voice Over.

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 20]
```diff
@@ -203,174 +550,348 @@
...
+            <li>
+              SR keyboard shortcuts: VO and NVDA and Orca.
+            </li>
+          </ul>
+          A few days before the workshop, I will send you these guides for you to practice.
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="Is there any discounts available?">
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:859]: <Accordion summary="Are there any discounts available?">

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 21]
```diff
@@ -203,174 +550,348 @@
...
+          The workshop will be in Early Bird price for a week. After that, there won’t be discounts available. I understand that the cost of this workshop can be too high for some people. If you are a student you can reach out to me.
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:860]: The workshop will have an Early Bird price for a week. After that, there won’t be more discounts available. Regardless, I understand that the cost of this workshop can be too high for some people, and if you are a student you can reach out to me.

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 22]
```diff
@@ -203,174 +550,348 @@
...
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="Can I buy a ticket to my team?">
+          You can but keep it mind that 1 ticket is for 1
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:866]: You can but keep in mind that 1 ticket is for 1

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 23]
```diff
@@ -203,174 +550,348 @@
...
+      </li>
+
+      <li>
+        <Accordion summary="Is there any discounts available?">
+          The workshop will be in Early Bird price for a week. After that, there won’t be discounts available. I understand that the cost of this workshop can be too high for some people. If you are a student you can reach out to me.
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="Can I buy a ticket to my team?">
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:865]: <Accordion summary="Can I buy a ticket for my team?">

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 24]
```diff
@@ -203,174 +550,348 @@
...
+          You can but keep it mind that 1 ticket is for 1 
+            <!-- TODO CLARIFY THIS -->
+          person/seat. Besides these public events, I also provide customised trainings. You can contact me to discuss private sessions options.
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:868]: person/seat. 
          Besides these public events, I also provide custom private sessions. Feel free to contact me to discuss this.

(trainings é uma palavra tricky pq em british english não é aceite, mas em american sim. eu acho que é melhor não usar for clarity, como é workshop internacional)

File: src/routes/workshop-a11y-fundamentals.svelte [hunk 25]
```diff
@@ -203,174 +550,348 @@
...
+
+      <li>
+        <Accordion summary="Can I ask you questions after the workshop?">
+          Of course! During the workshop, I'll invite to join a Discord community with other attendees to clarify any question.
+        </Accordion>
+      </li>
+
+      <li>
+        <Accordion summary="Is there a Code of Conduct?">
+          Yes, mine is pretty simple: Be kind and treat each other with respect and understanding. There’s zero tolerance for unkind behavior.
```
Comment (marianarpdf) [inline src/routes/workshop-a11y-fundamentals.svelte:880]: Yes, and it's pretty simple: be kind and treat each other with respect and understanding. There’s zero tolerance for unkind behavior.

File: src/routes/workshop-a11y-fundamentals.svelte (hunk 11 above)
Comment (sandrina-p) [reply src/routes/workshop-a11y-fundamentals.svelte:711]: > The topics will be explained with beginners in mind, and they will get more advanced as we go through them.

yah os topicos ficam mais avançados. gotcha.

File: src/routes/workshop-a11y-fundamentals.svelte (hunk 24 above)
Comment (sandrina-p) [reply src/routes/workshop-a11y-fundamentals.svelte:868]: Uh TIL. por acaso vejo tanto training como workshop e nunca soube a diferença. now I do.

**Pull request merged**